### ISSUE TITLE
🐞 Accessibility Request in Mac Sonama 14.5

### ISSUE THREAD

**Issue by cloofsiu** [MrKai77/Loop#476]
### Bug Description

🐞 Accessibility Request in Mac Sonama 14.5 does not work even I relaunchand reinstall the software. 

### Steps To Reproduce

/

### Expected Behavior

/

### Actual Behavior

/

### MacOS Version

Sonama 14.5

### Loop Version

Version 1.0.0 (1030)

### Final Checks

- [X] My issue is written in English
- [X] My issue title is descriptive
- [X] This is a single bug (multiple bugs should be reported individually)

**Comment by SenpaiHunters** [comment]
Check #418

**Comment by cloofsiu** [comment]
The steps I took:

Quit Loop
Remove Loop from Settings -> Privacy & Security -> Accessibility using the - button
Launch Loop (did not get Accessibility popup)
Add Loop to Accessibility settings using the + button


Problem solved. Thanks a lot!
